### PR TITLE
Fair 304 Accessibility Statement

### PR DESCRIPTION
**Pull request by sjose012**
**What changes does this PR bring?**

**Definition of Done**
The following need to be checked off as they are complete as part of the
PR process. If you are unable to mark off an item as complete, then state
in the comments why this is the case.
- [x] Have you built the code locally and confirmed its working?
- [x] Is the build confirmed to be passing on CI?
- [ ] Have you added enough relevant unit tests for your change?
- [ ] Have you added enough relevant E2E tests for your change?
- [ ] Have you updated any relevant documentation as part of this change?
- [ ] Has this change been tested against the Acceptance Criteria?
- [ ] Have you considered how this will be deployed in SIT/Staging/Production?

### CODE REVIEW

File: src/app/accessibility/index.njk [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+	<div class="govuk-grid-column-two-thirds">
+  <div class="govuk-notification-banner govuk-notification-banner--success" id="preference_changed" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner" tabindex="-1" hidden>
+  <div class="govuk-notification-banner__header">
+    <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+      Success
+    </h2>
+  </div>
+  <div class="govuk-notification-banner__content">
+    <p class="govuk-notification-banner__heading">
+      You’ve set your cookie preferences. <a class="govuk-notification-banner__link" href="javascript:history.back()">Go back to the page you were looking at</a>.
```
Comment (ame-test) [inline src/app/accessibility/index.njk:30]: Cookie stuff is already in the `base-template.njk`.
*Should the banner be defined here or not?

**Comment by ame-test** [comment]
Question (optional): How does the accessibility statement look on mobile?

**Comment by sjose012** [comment]
> Question (optional): How does the accessibility statement look on mobile?

UI is responsive one so it fits with the mobile screen.
Screenshot 2023-12-13 at 12 00 46

**Review by ame-test — APPROVED** [review]
looks good,

File: src/app/accessibility/index.njk (hunk 1 above)
Comment (sjose012) [reply src/app/accessibility/index.njk]: That is correct, I take the wrong file  for reference. Updated the code

**Comment by sjose012** [comment]
[FAIR-304]Accessibility Statement

**Pull request merged**